modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_ listed in the Additional resources section.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. To verify that your oc client version will support the `--to-multi-arch` command, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --help
+----
+* You must verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:
```

Comment:
This should go below the next command i.e. `oc adm upgrade`

Reply:
I've removed the "$ oc adm upgrade --help" command and related sentence. Is there another change you're requesting?

Reply:
Right, `You must verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:` is confusing because we do not see `RetrievedUpdates=True` in the output of `oc adm upgrade --help`

Reply:
got it :)